content/en/docs/Overview/_index.md

```diff
@@ -7,3 +7,30 @@ description: >
 ---
 
 Apache Parquet is a columnar storage format available to any project in the Hadoop ecosystem, regardless of the choice of data processing framework, data model or programming language.
+
+This documentation contains information about both the [parquet-mr](https://github.com/apache/parquet-mr) and [parquet-format](https://github.com/apache/parquet-format) projects. 
+
+
+### Parquet Format
+
+The "Parquet Format" project hosts the official specification of the Parquet file format, defining how data is structured and stored. This specification, along with Thrift metadata definitions and other crucial components, is essential for developers to effectively read and write Parquet files. The parquet-format project specifically contains the format specifications needed to understand and properly utilize Parquet files.
+
+As a repository focused on specification, the parquet-format repository does not contain source code. 
+
+
+### Parquet MR 
+
+The parquet-mr GitHub repository is part of the Apache Parquet project and specifically focuses on providing Java tools for handling the Parquet file format within the Hadoop ecosystem. Essentially, this repository includes all the necessary Java libraries and modules that allow developers to read and write Parquet files.
+
+ Parquet MR can be thought of the a "reference" implementation of parquet-format. There are a number of other Parquet Format implementations, such as [parquet-cpp](https://github.com/apache/parquet-cpp) and [parquet rust](https://github.com/apache/arrow-rs/blob/master/parquet/README.md). 
```

Comment:
Was this space intentional?
```suggestion
Parquet MR can be seen as the "reference" implementation of parquet-format. There are a number of other Parquet Format implementations, such as [parquet-cpp](https://github.com/apache/parquet-cpp) and [parquet rust](https://github.com/apache/arrow-rs/blob/master/parquet/README.md). 
```
Slight suggestion on the wording.

Reply:
Is parquet-mr actually the reference implementation, or would implementation in say parquet-cpp and parquet-rs be sufficient for standardisation of a feature? Are there already examples of this? Perhaps it is beyond the scope of this issue, but I think the relationship between parquet-mr and parquet the specification is at the core of this issue, and the broader governance ambiguity surrounding parquet.

Reply:
For the record: parquet-mr and cudf were used as the two different PoC implementations in the vote for format change https://issues.apache.org/jira/browse/PARQUET-2261.

Reply:
I'll tweak the wording - haha you can probably see I was going back and forth about whether or not to call it "a" reference implementation or "the" reference implementation. also h/t to @gszadovszky who's wording I borrowed for this.

Reply:
To follow up -- we are discussing reference implementations on the mailing list: https://lists.apache.org/thread/f9379yx0lf5gtpkgyv922pvowtzy4kmm